src/pages/saksbehandling/revurdering/bosituasjon/BosituasjonFormPage.tsx

```diff
@@ -0,0 +1,238 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Heading, Label, Textarea } from '@navikt/ds-react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Control, Controller, useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+
+import ApiErrorAlert from '~components/apiErrorAlert/ApiErrorAlert';
+import Feiloppsummering from '~components/feiloppsummering/Feiloppsummering';
+import { FnrInput } from '~components/FnrInput/FnrInput';
+import { BooleanRadioGroup } from '~components/formElements/FormElements';
+import { OppsummeringPar } from '~components/revurdering/oppsummering/oppsummeringspar/Oppsummeringspar';
+import ToKolonner from '~components/toKolonner/ToKolonner';
+import { lagreBosituasjonsgrunnlag } from '~features/revurdering/revurderingActions';
+import sharedMessages from '~features/revurdering/sharedMessages-nb';
+import { useAsyncActionCreator } from '~lib/hooks';
+import { MessageFormatter, useI18n } from '~lib/i18n';
+import { Nullable } from '~lib/types';
+import { hookFormErrorsTilFeiloppsummering } from '~lib/validering';
+import { Bosituasjon } from '~types/grunnlagsdataOgVilkårsvurderinger/bosituasjon/Bosituasjongrunnlag';
+import { RevurderingStegProps } from '~types/Revurdering';
+import * as DateUtils from '~utils/date/dateUtils';
+
+import { RevurderingBunnknapper } from '../bunnknapper/RevurderingBunnknapper';
+import sharedStyles from '../revurdering.module.less';
+import RevurderingsperiodeHeader from '../revurderingsperiodeheader/RevurderingsperiodeHeader';
+import UtfallSomIkkeStøttes from '../utfallSomIkkeStøttes/UtfallSomIkkeStøttes';
+
+import { BosituasjonFormData, bosituasjonFormValidation, getDefaultValues } from './bosituasjonForm';
+import messages from './bosituasjonForm-nb';
+import styles from './bosituasjonForm.module.less';
+
+const BosituasjonFormPage = (props: RevurderingStegProps) => {
+    const { formatMessage } = useI18n({ messages: { ...messages, ...sharedMessages } });
+    const [epsAlder, setEpsAlder] = useState<Nullable<number>>(null);
```

Comment:
Det kommer ikke helt godt frem her, men dette er egentlig en fil som er renamet, og ikke ny.
Men kunne vi sjekket alderen på fødselsnummeret selv, i steden for å få det som en del av responsen (`/person/søk`)? Da hadde vi sluttet denne staten. Man kan argumentere for at det ikke er single source of truth, når man både har fødselsnummer og alder :)